app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/roomcolumnviews/RoomColumnViewCache.kt

```diff
@@ -0,0 +1,112 @@
+package nerd.tuxmobil.fahrplan.congress.schedule.roomcolumnviews
+
+import android.content.Context
+import android.widget.LinearLayout
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.LayoutParams
+import nerd.tuxmobil.fahrplan.congress.models.RoomData
+import nerd.tuxmobil.fahrplan.congress.schedule.*
+import nerd.tuxmobil.fahrplan.congress.schedule.SessionViewColumnAdapter
+import nerd.tuxmobil.fahrplan.congress.schedule.SessionViewDrawer
+import nerd.tuxmobil.fahrplan.congress.schedule.SessionViewEventsHandler
+import nerd.tuxmobil.fahrplan.congress.schedule.roomcolumnviews.RoomColumnViewCache.RoomColumnView
+
+internal interface IRoomColumnViewCache {
```

Comment:
Do we need the additional layer of indirection introduced by `RoomColumnViewProvider`?  I thought we could just rename `IRoomColumnViewCache` to `RoomColumnViewProvider` and drop `RoomColumnViewProvider` itself.

BTW: No personal preference - just for consistency: the `I` prefix isn't used in this project. I would therefore prefer the suggested naming.